paymentsheet/src/main/java/com/stripe/android/paymentelement/EmbeddedPaymentElementKtx.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.paymentelement
+
+import androidx.annotation.RestrictTo
+import androidx.compose.runtime.Composable
+import androidx.lifecycle.viewmodel.compose.LocalViewModelStoreOwner
+
+@ExperimentalEmbeddedPaymentElementApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+@Composable
+fun rememberEmbeddedPaymentElement(): EmbeddedPaymentElement {
+    val viewModelStoreOwner = requireNotNull(LocalViewModelStoreOwner.current) {
+        "EmbeddedPaymentElement must have a ViewModelStoreOwner."
+    }
+    return EmbeddedPaymentElement.create(
```

Comment:
Should we wrap this in Compose's `remember` as well so that instances aren't created on every render?

Reply:
🤦 Thanks!